keyboards/giabalanaipico/info.json

```diff
@@ -0,0 +1,219 @@
+{
+    "keyboard_name": "giabalanaipico",
+    "manufacturer": "3araht",
+    "url": "https://github.com/3araht",
+    "maintainer": "3araht",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "features": {
+        "midi": true,
+        "extrakey": true,
+        "encoder": true,
+        "bootmagic": false,
+        "console": false,
+        "mousekey": false,
+        "nkro": false,
+        "rgblight": false,
+        "audio": false
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": [ "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16"  ],
+        "rows": [ "GP4",  "GP5",  "GP7",  "GP9",  "GP10", "GP11", "GP12", "GP13", "GP14", "GP15" ]
+    },
+    "debounce": 5,
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0xF4C0",
+        "device_version": "0.0.1"
+    },
+    "ws2812": {
+        "pin": "GP6",
+        "driver": "vendor"
+    },
+    "encoder": {
+        "rotary": []
+    },
```

Comment:
not used/non-functional
```suggestion
```

Reply:
This is needed to avoid the following error:
```
quantum/encoder.h:35:56: error: 'ENCODERS_PAD_A' undeclared here (not in a function); did you mean 'ENCODERS_PAD_A_RIGHT'?
 #        define NUM_ENCODERS_LEFT ARRAY_SIZE(((pin_t[])ENCODERS_PAD_A))
```

Reply:
If encoders are present on the right, but not the left, then this is correct.